src/components/batch-selection/batch-selection.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import BatchSelection from '.';
+import IconButton from '../icon-button';
+import Icon from '../icon';
+
+<Meta title="Design System/Batch Selection" />
+
+# Batch Selection
+
+
+### Basic usage:
+
+<Preview>
+  <Story name="transparent" parameters={{ info: { disable: true }}} >
+    <BatchSelection selectedCount={3}>
+      <IconButton onAction={ () => {} }><Icon type='csv' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='bin' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='pdf' /></IconButton>
+    </BatchSelection>
+  </Story>
+</Preview>
+
+### on dark background:
+
+<Preview>
+  <Story name="basic" parameters={{ info: { disable: true }}} >
+    <BatchSelection selectedCount={3} colorTheme='dark'>
+      <IconButton onAction={ () => {} }><Icon type='csv' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='bin' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='pdf' /></IconButton>
+    </BatchSelection>
+  </Story>
+</Preview>
+
+### On light background:
+
+<Preview>
+  <Story name="light" parameters={{ info: { disable: true }}} >
+    <BatchSelection selectedCount={3} colorTheme='light'>
+      <IconButton onAction={ () => {} }><Icon type='csv' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='bin' /></IconButton>
+      <IconButton onAction={ () => {} }><Icon type='pdf' /></IconButton>
+    </BatchSelection>
+  </Story>
+</Preview>
+
+### Disabled:
+
+<Preview>
+  <Story name="disabled" parameters={{ info: { disable: true }}} >
```

Comment:
I noticed when looking at this demo that it has the default cursor when disabled. Typically we have used the `not-allowed` one. Is this intentional?

Reply:
my bad, i forgot about that. Fixed.